doc/modules/grid_search.rst

```diff
@@ -1,10 +1,10 @@
-.. currentmodule:: sklearn.grid_search
+.. currentmodule:: sklearn.model_selection
 
 .. _grid_search:
 
-===============================================
-Grid Search: Searching for estimator parameters
-===============================================
+==========================================
+Search: Searching for estimator parameters
```

Comment:
Your built at http://rvraghav93.github.io/scikit-learn/doc/_build/html/stable/modules/grid_search.html doesn't seem to be using this branch.